URBNSwiftyConvenience/Classes/LayoutConvenience.swift

```diff
@@ -135,3 +135,55 @@ public extension UIStackView {
         }
     }
 }
+
+// TODO: Refactor out when we move to swifty 4
+//       Also we could adopt add & subtract capability similar to - https://useyourloaf.com/blog/easier-swift-layout-priorities/
+@available(swift, obsoleted: 4.0)
+public struct ConstraintPriority: RawRepresentable {
+    
+    public static let defaultLow: ConstraintPriority = ConstraintPriority(UILayoutPriorityDefaultLow)
+    public static let defaultHigh: ConstraintPriority = ConstraintPriority(UILayoutPriorityDefaultHigh)
+    public static let required: ConstraintPriority = ConstraintPriority(UILayoutPriorityRequired)
+    public static let fittingSizeLevel: ConstraintPriority = ConstraintPriority(UILayoutPriorityFittingSizeLevel)
+
+    public var rawValue: Float
+
+    public init(_ rawValue: Float) {
+        self.rawValue = rawValue
+    }
+    
+    public init(rawValue: Float) {
```

Comment:
Why the two inits, they are the same

Reply:
Was matching UILayoutPriority. One is needed with RawRepresentable.

Reply:
See https://developer.apple.com/documentation/uikit/uilayoutpriority